doc/source/serve/production.md

```diff
@@ -296,18 +284,19 @@ The message `Sent deploy request successfully!` means:
 * It will start a new Serve application if one hasn't already started.
 * The Serve application will deploy the deployments from your deployment graph, updated with the configurations from your config file.
 
-It does **not** mean that your Serve application, including your deployments, has already started running successfully. This happens asynchronously as the Ray cluster attempts to update itself to match the settings from your config file. Check out the [next section](serve-in-production-inspecting) to learn more about how to inspect your deployments.
-
-### Adding a Runtime Environment
+It does **not** mean that your Serve application, including your deployments, has already started running successfully. This happens asynchronously as the Ray cluster attempts to update itself to match the settings from your config file. Check out the [next section](serve-in-production-inspecting) to learn more about how to get the current status.
 
-If you start Ray and deploy your deployment graph from a directory that doesn't contain the graph code, your deployments will fail to run. This happens because your import path is generally location-dependent. For example, the import path `fruit.deployment_graph` assumes the current directory contains the `fruit.py` module, which contains a `deployment_graph` object.
+### Adding a runtime environment
 
-To make your config file location-independent, you can push your deployment graph code to [a remote repository and add that repository to your config file's `runtime_env` field](remote-uris). When Serve runs your deployment graph, it will pull the code from the remote repository rather than use a local copy. **This is a best practice** because it lets you deploy your config file from any machine in any directory and share the file with other developers, making it a more standalone artifact.
+The import path (e.g., `fruit:deployment_graph`) must be importable by Serve at runtime.
+When running locally, this might be in your current working directory.
+However, when running on a cluster you also need to make sure the path is importable.
+You can achieve this either by building the code into the cluster's container image or by using a `runtime_env` with a [remote URI](remote-uris) that hosts the code in remote storage.
```

Comment:
> You can achieve this either by building the code into the cluster's container image

Is there documentation about a `cluster's container image` that we can link to? New Ray Serve users might not know how to create a Ray cluster that uses container images.

Reply:
added a link